<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="USER" failures="0" tests="3" name="invalid_login" time="19.767" errors="0" timestamp="2024-07-26T23:58:59 GMT+02:00" skipped="0">
  <testcase classname="invalid_login" name="Invalid_Login_1" time="6.646">
    <system-out>
      <![CDATA[-- Trying Enter Invalid Credentials
-- Click Login Button
-- Pop-up window show with error text]]>
    </system-out>
  </testcase> <!-- Invalid_Login_1 -->
  <testcase classname="invalid_login" name="Invalid_Login_2" time="6.607">
    <system-out>
      <![CDATA[-- Trying Enter Credentials
-- Click Login Button
-- Pop-up window show with error text]]>
    </system-out>
  </testcase> <!-- Invalid_Login_2 -->
  <testcase classname="invalid_login" name="Invalid_Login_3" time="6.514">
    <system-out>
      <![CDATA[-- Trying Enter Credentials
-- Click Login Button
-- Pop-up window show with error text]]>
    </system-out>
  </testcase> <!-- Invalid_Login_3 -->
  <system-out>
    <![CDATA[-- Open Base URL
-- Trying Enter Credentials
-- Click Login Button
-- Reaching HomePage
-- Manager ID is shown in o/p
-- Passed Test case
-- Open Base URL
-- Trying Enter Invalid Credentials
-- Click Login Button
-- Pop-up window show with error text
-- Open Base URL
-- Trying Enter Credentials
-- Click Login Button
-- Pop-up window show with error text
-- Open Base URL
-- Trying Enter Credentials
-- Click Login Button
-- Pop-up window show with error text]]>
  </system-out>
</testsuite> <!-- invalid_login -->
